docs/source/open_science/GIN-repositories.md

```diff
@@ -0,0 +1,357 @@
+   ```
+
+   This command will create new files that were added remotely, delete files that were removed, and update files that were changed.
+
+   With the `--content` flag, it optionally downloads the content of all files in the repository. If `--content` is not specified, new files are added as empty placeholders.
+
+2. The content of individual files can be retrieved using:
+   ```
+   $ gin get-content <filename>
+   ```
+   and later removed with:
+   ```
+   $ gin remove-content <filename>
+   ```
+
+   See [the GIN docs](https://gin.g-node.org/G-Node/Info/wiki/GIN+CLI+Help#download-all-new-information-from-a-remote-repository) for further details.
+
+### To download the data programmatically in your Python code:
+
+We recommend using [pooch](https://www.fatiando.org/pooch/latest/index.html) to easily download data from the GIN repo's URL. Pooch also has some other nice functionalities like caching the downloaded data, verifying cryptographic hashes or unzipping files upon download.
+
+## File locking
+
+[File locking](https://gin.g-node.org/G-Node/Info/wiki/GIN+CLI+Usage+Tutorial#file-locking) is an important point in GIN and git-annex repositories which surprisingly comes up quite late in the GIN docs. Below, are the main ideas behind this.
+
+- Files in a GIN repo can be _locked_ or _unlocked_.
+
+- The lock state relates to the nature of the placeholder files we get in the working directory when we fetch the remote repository via `gin get <remote-repository-location>`:
+
+  - **on Unix-like systems** (Mac, Ubuntu):
+    - if a file is _locked_, its corresponding placeholder file will be a _symlink_. These symlinks point to the annexed content (under `.git/annex/objects`). With the symlinks we can open the file but not modify it.
+    - If a file is _unlocked_, the placeholder file in the working directory is an _ASCII text file with a path_. The path is approximately where the content of the file will be downloaded to when we request it.
+  - **on Windows**:
+    - if a file is _locked_, the placeholder file is a _plain text file_ pointing to the content in the git annex.
+    - If a file is _unlocked_, the behaviour should be the same as in Unix-like systems (I haven't tested this directly).
+
+- The lock state of a file is _persistent_. This means that if I clone a GIN repo whose files are unlocked, I lock them in my local copy, and then upload that to the GIN server, the next time someone clones from the GIN repo the files they fetch will be locked.
+
+- Unlocked files can be edited. If the data is unlocked and the full content of the dataset is downloaded locally, the file in the working directory has content, and so does its copy under git annex.
+
+:::{caution}
+Notice that if we download locally the content of unlocked, the disk usage of the files checked into the repo doubles, because the content exists both in the working directory and under the git annex. But in exchange users can modify and revert files to previous commits.
```

Comment:
```suggestion
Note that if we download the contents of unlocked files locally, the disk usage of the files checked into the repo doubles, because the content exists both in the working directory and under the git annex. But in exchange users can modify and revert files to previous commits.
```
this sounds more natural to me, but I am not a native speaker.